migration/2021-03-19-communes-collection.js

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+require('dotenv').config()
+
+const mongo = require('../lib/util/mongo')
+
+async function migrateCommune() {
+  const basesLocales = await mongo.db.collection('bases_locales').find().toArray()
+
+  basesLocales.forEach(async ({_id, communes}) => {
```

Comment:
Il faut utiliser `for ... of` car forEach fait perdre le contrôle sur l'appel asynchrone.
`Promise.all` est aussi une option, mais trop agressif pour MongoDB.
Il y a aussi bluebird.